script/run-isolated-tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Get the list of subdirectories that contain apps and
+# extract substrings from the paths for the following regex.
+APP_SUBPATHS="$(find src -name manifest.json | sed -E 's/\/manifest.json//g' | xargs | sed 's/ /|/g')"
+
+# Match changed files to the app paths determined above
+FILES_CHANGED="$(git diff --name-only master)"
+APP_SUBPATHS_CHANGED="$(echo $FILES_CHANGED | grep -oE "($APP_SUBPATHS)")"
+NUM_APPS_CHANGED=$(echo "$APP_SUBPATHS_CHANGED" | wc -l)
```

Comment:
If this runs on a PR branch and master is ahead of it, then we end up with `APP_SUBPPATHS_CHANGED` that doesn't contain app subpaths that the PR changed, right?

Reply:
I think it still does as long as it differs from what exists on `master`.